doc/nrf/ug_radio_coex.rst

```diff
@@ -129,18 +193,18 @@ Wi-Fi Coexistence implementations
 
 The following CX implementations are available:
 
-* :ref:`ug_radio_cx_generic_3pin_impl`
+* :ref:`ug_radio_cx_nrf700x_impl`
 * :ref:`ug_radio_cx_thread_impl`
 
 Each implementation is a plugin composed of one or more C source files interfacing with the MPSL API, which, in turn, communicates with the SR protocol drivers.
 
 When one of the CX variants is enabled, the driver requests access to the RF medium from the PTA and informs it about the operation it is about to perform.
 It also reacts properly to the information from the PTA that access to the RF medium was either granted or denied.
 
-.. _ug_radio_cx_generic_3pin_impl:
+.. _ug_radio_cx_nrf700x_impl:
 
-Generic 3-pin interface implementation
-======================================
+nRF700x interface implementation
+================================
```

Comment:
Again, if https://github.com/nrfconnect/sdk-nrf/pull/8750/files#r972850052 is addressed the pin names
below (REQUEST, PRIORITY, GRANT) will possible need to be renamed.